src/index.js

```diff
@@ -497,7 +514,9 @@ async function run (input, output, { puppeteerConfig = {}, quiet = false, output
       info('Generating single mermaid chart')
       browser = await puppeteer.launch(puppeteerConfig)
       const data = await parseMMD(browser, definition, outputFormat, parseMMDOptions)
-      await fs.promises.writeFile(output, data)
+      await output !== '/dev/stdout'
+        ? fs.promises.writeFile(output, data)
+        : process.stdout.write(data)
```

Comment:
Nice work on using `process.stdout.write`!

I think this might actually make this code compatible with Windows too (and other non-Linux OSes that don't support `/dev/stdout`).

Reply:
Thanks!